user-service/middlewares/redis.js

```diff
@@ -0,0 +1,69 @@
+import { createClient } from 'redis';
+
+const client = createClient({
+    url: 'redis://localhost:6379' // Adjust the URL if using Docker or a remote instance
```

Comment:
Should we use an environment variable here to allow us to switch endpoints during deployment?